templates/certified/components/component-details.html

```diff
@@ -0,0 +1,34 @@
+{% extends "templates/base.html" %}
+
+{% block title %}Ubuntu certified component: {{ component.vendor_name }} {{ component.model }}{% endblock %}
+
+{% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block outer_content %}
+
+<div class="row">
+  {% with 
+    vendor_name=component.vendor_name,
+    component_name=component.model
+  %}
+    {% include "/certified/components/_hero-section.html" %}
+  {% endwith %}
+
+  {% with
+    lts_releases=component.lts_releases
+  %}
+    {% include "/certified/components/_certified-releases.html" %}
+  {% endwith %}
+  <hr/>
```

Comment:
Since these hrs are outside each section row, we can used the fixed-width class to align it correctly to the grid
```suggestion
  <hr class="p-rule is-fixed-width" />
```